src/main/java/io/jenkins/pluginhealth/scoring/probes/Probe.java

```diff
@@ -36,7 +36,7 @@ public abstract class Probe {
      * Starts the analyze on a plugin.
      * Should only be called by the {@link ProbeEngine#run()} method.
      *
-     * @param plugin the plugin on which to perform the analyze
+     * @param plugin  the plugin on which to perform the analyze
```

Comment:
I'd remove this additional space (same below)

Reply:
It's the formatting so that the definitions start on the same column for the parameters.